React/Views/RCTModalHostView.m

```diff
@@ -61,10 +63,11 @@ - (void)notifyForBoundsChange:(CGRect)newBounds
 
 - (void)notifyForOrientationChange
 {
+#if !TARGET_OS_TV
   if (!_onOrientationChange) {
     return;
   }
-
+  
```

Comment:
Weird I committed the change to remove the whitespace, but the diff doesn't show it....